FlowCrypt/src/main/java/com/flowcrypt/email/jetpack/viewmodel/ClientConfigurationViewModel.kt

```diff
@@ -29,17 +29,18 @@ class ClientConfigurationViewModel(application: Application) : BaseAndroidViewMo
   val clientConfigurationLiveData: MutableLiveData<Result<ClientConfigurationResponse>> =
     MutableLiveData(Result.none())
 
-  fun fetchClientConfiguration(idToken: String, fesUrl: String? = null) {
+  fun fetchClientConfiguration(idToken: String, baseFesUrlPath: String, domain: String) {
     viewModelScope.launch {
       val context: Context = getApplication()
       clientConfigurationLiveData.value =
         Result.loading(progressMsg = context.getString(R.string.loading_domain_rules))
 
       try {
-        clientConfigurationLiveData.value = repository.getClientConfiguration(
+        clientConfigurationLiveData.value = repository.getClientConfigurationFromFes(
```

Comment:
The `repository` (or what I'd prefer to call it, `apiClient`) has calls to all different services all in a single class, making it not so clear which service is called for what. This particular call is `getClientConfigurationFromFes` which is clear, but the other ones are not, this it lacks consistency.

If you'd like them all in one class, then we should edit all of the method names to make it clear which service is being called. For consistency we could prepent the name of the service. So it would be:

```
apiClient.fesGetClientConfiguration
apiClient.attesterPostWelcomeMessage
apiClient.backendPostHelpFeedback
```

I'd prefer the following, but would it complicate it for you?

```
apiClient.fes.getClientConfiguration
apiClient.attester.postWelcomeMessage
apiClient.backend.postHelpFeedback
```

and so on. Then it's 100% clear which server is being called.

Reply:
It's a good comment. It will make the code clearer for external devs. I will check what I can do.

Reply:
I think for you, too. Because with so many servers, after some time, one can forget what request went where. Which can lead to bugs when refactored.

Reply:
I've added new changes. I hope now it looks better.